### ISSUE TITLE
Dynamically load or automatically update schismtracker.org to point to latest release

### ISSUE THREAD

**Issue by jangler** [schismtracker/schismtracker#110]
Branched off from #40. Properly this should be a [schismtracker.github.io](https://github.com/schismtracker/schismtracker.github.io) issue, but practically it makes more sense to have it here.

**Comment by asbjornu** [comment]
I think this can be achieved by changing from the current static `index.html` file to [Jekyll](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/). I might have a whack at it as a PR against that repository.

**Comment by jangler** [comment]
Yeah, I was hoping to avoid Jekyll, but that might be the sanest way to do it. The alternative would be JavaScript, which I'm not any more keen on including, so might as well do it server-side.
